ontology/yaml/resources/HVAC/entity_types/VAV.yaml

```diff
@@ -47,6 +47,15 @@ VAV_SD_DSP:
   - SD
   - DSP
 
+VAV_SD_HLSAFS:
+  guid: "c7ea23bd-8511-42be-be58-aa6d3739e93d"
+  description: "Single duct VAV with flowrate monitoring."
```

Comment:
Because the HLSAFS abstract type uses flowrate setpoints, please update the description to "Single duct VAV with flowrate control."

Reply:
@shambergoldstein - thank you, I've updated description. Ciaran